quadriga/src/test/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImplTest.java

```diff
@@ -132,6 +140,25 @@ public boolean matches(Collection arg0) {
         managerToTest.deleteCollection(id.toString());
     }
     
+    @Test
+    public void test_deleteCollection_archived() throws CollectionNotFoundException, InvalidObjectIdException {
+        String name = "name";
+        String desc = "description";
+        Collection collection = new Collection();
+        ObjectId id = new ObjectId();
+        collection.setId(id);
+        collection.setName(name);
+        collection.setDescription(desc);
+        Collection response = new Collection();
```

Comment:
if we change the code to do something to the collection after it was save, then this test will fail if you just return an empty collection object. Maybe just return `collection` in line 155?